insights/parsers/multipath_conf.py

```diff
@@ -130,7 +129,46 @@ def _create_parser(cls):
         return my_conf
 
     def parse_content(self, content):
-        self.data = MultipathConf._create_parser().parseString("\n".join(content))[0].asDict()
+        self.data = MultipathConfParser._create_parser().parseString("\n".join(content))[0].asDict()
```

Comment:
AFAIK, the output of the command `/bin/lsinitrd -f /etc/multipath.conf` could be empty when there is no such file in the `initramfs`, I think we'd better skip this case by ignoring an Exception for empty content, how do you think?